src/nav/IconUnreadConversations.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
```

Comment:
Also, we can have a general component `TabBarNavButton` for this. Though we will have to have one special case for `Streams` icon.